src/sections/Hero/style.module.scss

```diff
@@ -1,59 +1,62 @@
 @use "src/styles/vars.scss" as vars;
 
-@import url("https://fonts.googleapis.com/css2?family=Rubik&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
 
 .heroContainer {
-  background-image: linear-gradient(180deg, #100b44 15%, #a78590 98%);
-  min-height: 100vh;
-  font-family: Rubik, sans-serif;
+    background-image: linear-gradient(180deg, #100b44 15%, #a78590 98%);
+    min-height: 100vh;
+    font-family: Rubik, sans-serif;
+    padding-top: 10px;
 
-  .heroTitle {
-    position: absolute;
-    top: 18%;
-    z-index: 1;
-  }
+    .heroTitle{
+        position: absolute;
+        top: 18%;
+        z-index: 1;
+    }
+
+    .heroPlanet{
+        max-width: 60vh;
+        max-height: 60vh;
+        position: absolute;
+        top: 60%;
+        bottom: 0;
+        right: 0;
+        z-index: 1;
+    }
 
-  .heroPlanet {
-    max-width: 60vh;
-    max-height: 60vh;
-    position: absolute;
-    top: 60%;
-    bottom: 0;
-    right: 0;
-    z-index: 1;
-  }
+   .heroDescription{
+        display: flex;
+        text-align: center;
+        justify-content: center;
+        flex-direction: column;  
+        position: absolute;
+        height: 100vh;
+        width: 100vw;
+        margin-top: 11vw;
+        padding: 0 2rem;
+        z-index: 2;
+        
+        .heroSubtitle{
+            line-height: 100%;
+            margin-top: 6rem;
+            color: vars.$white;
+            font-size: 40px;
+            font-weight: bolder; 
+        }
 
-  .heroDescription {
-    display: flex;
-    text-align: center;
-    justify-content: center;
-    flex-direction: column;
-    position: absolute;
-    height: 100vh;
-    width: 100vw;
-    padding: 6rem 2rem 0 2rem;
-    z-index: 2;
+        .heroInfo{
+            color: vars.$white;
+            font-size: 20px;
+        }
 
-    .heroSubtitle {
-      line-height: 100%;
-      margin-top: 20vw;
-      color: vars.$white;
-      font-size: 40px;
-      font-weight: bolder;
     }
-
-    .heroInfo {
-      color: vars.$white;
-      font-size: 20px;
+    .registerButton{
+        color: white;
+        font-weight: bolder;
+        background: #110F47;
```

Comment:
Nitpick but let's try to use color variables to make sure we're referencing the exact color everywhere and can easily change when we need to